docs/src/advanced.md

```diff
@@ -0,0 +1,167 @@
+# Advanced features
+
+## Multi-precision
+
+All algorithms can run in various precisions modes: `Float16, Float32, Float64, BigFloat` and also for rationals based on various integer types `Int32, Int64, BigInt` (see e.g., the approximate Carathéodory example)
+
+## Step size computation
+
+For all Frank-Wolfe algorithms, a step size must be determined to move from the current iterate to the next one.
+This step size can be determined by exact line search or any other rule represented by a subtype of [`FrankWolfe.LineSearchMethod`](@ref), which must implement [`FrankWolfe.line_search_wrapper`](@ref).
+
+Multiple line search and step size determination rules are already  available.
```

Comment:
maybe name some line search strategies already defined in the package

Reply:
Sure! I only restructured what was already there, adding new content is up to you ;)